### DISCUSSION TITLE
Status ein / an Unterschiede

### DISCUSSION THREAD

**Discussion by tofrie** [Zefau/ioBroker.jarvis#932] [Q&A] [answered]
Hi zusammen, kann mir jemand sagen warum rechts bei meinen Geräten im eingeschalteten Zustand mal an und mal ein steht? Es werden in allen Fällen Datenpunkte abgerufen welche nur mit true/false gesteuert werden.
Danke für eure Hilfe.
![8D96EA04-64CD-4116-994F-C324DA5517EF](https://user-images.githubusercontent.com/60553702/131795177-00c38052-a68d-4248-8067-57badf20dea8.png)

**Comment by Zefau** [comment, marked as answer]
`true`/`false` sollte nicht erscheinen. Was ist das für ein Gewerk und wie lautet der StateKey (also dein Bezeichner) ?

`an` ist Jarvis Standard. `ein` hast du eingestellt?

**Reply by tofrie** [reply]
Sorry, es war mein Fehler. Ich habe die Werte so definiert. Danke für deinen Hinweis 🙏
